tdrs-backend/tdpservice/stts/migrations/0006_alter_stt_filenames.py

```diff
@@ -47,7 +47,7 @@ def add_filenames(apps, schema_editor):
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('stts', '0005_stt_stt_code'),
```

Comment:
We shouldn't have to change the previous migrations. This will create problem for deployment.